lib/activerecord-bitemporal/scope.rb

```diff
@@ -353,14 +362,17 @@ def _#{column}_#{op}(datetime,**)
 
       # transaction_from <= datetime && datetime < transaction_to
       scope :transaction_at, -> (datetime) {
+        if ActiveRecord::Bitemporal.force_transaction_datetime?
+          datetime = ActiveRecord::Bitemporal.transaction_datetime
+        end
         datetime = Time.current if datetime.nil?
         transaction_from_lteq(datetime).transaction_to_gt(datetime).with_transaction_datetime
       }
       scope :ignore_transaction_datetime, -> {
         ignore_transaction_from.ignore_transaction_to.without_transaction_datetime
       }
       scope :except_transaction_datetime, -> {
-        except_transaction_from.except_transaction_to.without_transaction_datetime
+        except_transaction_from.except_transaction_to.tap { |relation| relation.bitemporal_value.except! :with_transaction_datetime }
       }
 
       scope :bitemporal_at, -> (datetime) {
```

Comment:
It seems that `bitemporal_at` only works for `ignore_valid_datetime` and doesn't affect `ignore_transaction_datetime`. Is this the intended behavior?

Reply:
Good catch :+1: 
I fixed https://github.com/kufu/activerecord-bitemporal/pull/82/commits/93e7e33eeaf51d74bb34297f165292ef902c79ca